packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -143,14 +143,14 @@ export default function MainFeedLayout({
   });
   const feedVersion = useFeature(feature.feedVersion);
   const searchVersion = useFeature(feature.searchVersion);
-  const hasCommentFeed = useFeature(feature.commentFeed);
   const { isUpvoted, isPopular, isSortableFeed } = useFeedName({ feedName });
-  const { shouldUseMobileFeedLayout, FeedPageLayoutComponent } =
-    useFeedLayout();
+  const {
+    shouldUseMobileFeedLayout,
+    FeedPageLayoutComponent,
+    shouldUseCommentFeedLayout,
```

Comment:
```suggestion
    shouldUseCommentFeedLayout,
    shouldUseMobileFeedLayout,
    FeedPageLayoutComponent,
```
Just a minor nitpick, personal preference and makes it easier to see that the `shouldUse...` are different, and not the same 😅

Reply:
Makes sense 👍